doc/api/fs.md

```diff
@@ -4390,7 +4415,7 @@ changes:
 -->
 
 * `fd` {integer}
-* `buffer` {Buffer|TypedArray|DataView|string|Object}
+* `buffer` {Buffer|TypedArray|DataView}
```

Comment:
We'd need a change entry here too: https://github.com/nodejs/node/blob/ae1f6db0886a540dcf33f27bb4f13c37e0c8899e/doc/api/fs.md#L4354-L4367

Reply:
Resolved in the same way as [filehandle.write](https://github.com/nodejs/node/pull/41677#discussion_r841283211): redundant changelog entry is removed, string-related changes are kept in `fs.write(fd, string, ...)` method.

Note: string version of this method *does* coerce Objects even after this PR. This is doc-only segregation.